blogs/ecs-ec2-crossvpc-with-tls/mesh.yml

```diff
@@ -0,0 +1,158 @@
+---
+AWSTemplateFormatVersion: "2010-09-09"
+Description: This stack deploys the AppMesh resources
+Resources:
+  AppMesh:
+    Type: "AWS::AppMesh::Mesh"
+    Properties:  
+      MeshName: "appmesh-workshop"
+      Spec:
+        EgressFilter:
+          Type: "DROP_ALL"
```

Comment:
Minor: you can omit this entirely, the default behavior is `DROP_ALL`.

Reply:
resolved